newIDE/electron-app/app/ServeFolder.js

```diff
@@ -27,6 +28,19 @@ module.exports = {
           // the hot-reloading built into the game engine is what should
           // be used - and the user can still reload manually on its browser.
           watch: [],
+          middleware: [
+            // Disable caching, as it can lead to older generated code being served
```

Comment:
```suggestion
            // Disable caching, as it can lead to older generated code being served
            // in case preview files are proxied through a CDN (see 
            // https://github.com/4ian/GDevelop/pull/6553 for example)
```